contrib/resources/translations/normalize.bat

```diff
@@ -0,0 +1,31 @@
+@ECHO OFF
```

Comment:
Is it possible to add a license header?

```bat
REM SPDX-License-Identifier: GPL-2.0-or-later
REM
REM Copyright (C) 2022-2023  The DOSBox Staging Team
```

@Kappa971 , are you the author? Maybe this can be a separate commit with your as the commit author. Alternatively, if @FeralChild64 helped or made adjustments, use co-author: attribution in the commit.

Reply:
Yes, I know this seems petty or unnecessary; this is more about eliminating ambiguity and unknowns.

Reply:
This is an old file, previously called encode.bat - it doesn't encode anything anymore, so I have renamed it. Unfortunately, there was enough changes that GIT stopped recognizing this as renamed file.

Reply:
License header added. @kcgen - not sure how to proceed with copyright in this case; add a co-author? I would prefer to keep everything in one commit, or else we will have a non-working script in some point of GIT history.

Reply:
ahh not a big deal; one commit is good.